cypress/page/login.page.ts

```diff
@@ -0,0 +1,25 @@
+class LoginPage {
+  private email: string;
+  private password: string;
+  private btnSubmit: string;
+
+  constructor() {
+    this.email= "#email"
+    this.password= "#passwd"
+    this.btnSubmit= "#SubmitLogin"
+  }
+
+  public addingLoginEmail(): void {
+    cy.get(this.email).type("aperdomobo@gmail.com")
+  }
+
+  public addingLoginPasswd(): void {
```

Comment:
addPass or typePass would be a better name :) The name of methods should contain verbs in infinitive mode.